src/Groups/pcgroup.jl

```diff
@@ -365,3 +364,87 @@ function pc_group(c::GAP_Collector)
   end
 end
 
+"""
+    letters(g::Union{PcGroupElem, SubPcGroupElem})
+
+Return the letters of `g` as a list of integers, each entry corresponding to
+a group generator.
```

Comment:
Note that we can also produce negative numbers: e.g. -3 means "inverse of 3rd generator". This should be explained, and perhaps an example added showing that. E.g. based on this:

```
julia> x = (gg[1]*gg[2]*gg[3])^-2
g1*g2^-2*g3^3
```

Perhaps also add something like this (and then mirror it in the other function)
> See also [`syllables`](@ref).

Reply:
I have added a small example with some brief explanation to `letters` for this. However I am unsure if the example is good as I was not able to get elements with negative exponents and test.

Reply:
For that you need an infinite group. E.g.
```
julia> g = dihedral_group(PosInf())
Pc group of infinite order

julia> g[1]^-3 * g[2]^-3
g1*g2^-3
```
or
```
julia> g = abelian_group(PcGroup, [5, 0])
Pc group of infinite order

julia> g[1]^-3 * g[2]^-3
g1^2*g2^-3
```